ush/test/README.md

```diff
@@ -1,5 +1,5 @@
 ## Test bufr_query mapping, Python converter script, and ioda configuration YAML in obsForge
-This is a prototype for testing BUFR to IODA conversion and is still evolving.
+This is a prototype for testing BUFR to NETCDF/IODA conversion and is still evolving.
```

Comment:
We will use this for ZARR as well... These could also be used in IODA backends too.